doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md

```diff
@@ -1,4 +1,6 @@
 #<a name="top"></a>NGSIGroupingInterceptor
+**IMPORTANT NOTE: from release 1.6.0, this feature is deprecated in favour of Name Mappings.**
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf